packages/service-library/src/servicelib/fastapi/long_running_tasks/_client.py

```diff
@@ -128,12 +130,9 @@ def _client_configuration(self) -> ClientConfiguration:
         output: ClientConfiguration = self.app.state.long_running_client_configuration
         return output
 
-    def _get_url(self, path: str) -> AnyHttpUrl:
-        output: AnyHttpUrl = parse_obj_as(
-            AnyHttpUrl,
-            f"{self._base_url}{self._client_configuration.router_prefix}{path}",
-        )
-        return output
+    def _get_url(self, path: str) -> str:
+        url = f"{self._base_url}{self._client_configuration.router_prefix}{path}"
+        return f"{AnyHttpUrlLegacyAdapter.validate_python(url)}"
```

Comment:
I again suggest this path, should be smoother to read
```suggestion
        any_http_url = AnyHttpUrlLegacyAdapter.validate_python(f"{self._base_url}{self._client_configuration.router_prefix}{path}")
        return f"{any_http_url}"
```